src/Route.elm

```diff
@@ -20,6 +22,12 @@ toString route =
         Index ->
             "/"
 
+        CaseStudy ->
+            "/case-study"
+
+        Resource ->
+            "/resource"
+
 
 routeParser : Parser (Route -> a) a
 routeParser =
```

Comment:
can't seem to add a comment to all the lines but we probably want to read the routes here.

```elm
routeParser : Parser (Route -> a) a
routeParser =
    oneOf
        [ map Index top
        , map Resource (s "resource")
        , map CaseStudy (s "case-study")
        ]
```

https://package.elm-lang.org/packages/elm/url/latest/Url-Parser

That should then update the `model.page` when we load.